### PR TITLE
add deltadore-tydom

### PR DESCRIPTION
**Pull request by CyrilP**
## Proposed change

add deltadore-tydom logos and incons

## Type of change

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [X] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: 
- Link to documentation pull request: 
- Link to integration documentation on our website: 

## Checklist

- [X] The added/replaced image(s) are **PNG**
- [X] Icon image size is 256x256px (`icon.png`)
- [X] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [X] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [X] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

**Review by frenck — APPROVED** [review]
Hi there @CyrilP 👋 

I've looked at the upstream repository, but noticed it only contains a blueprint copy, and no actual working integration.

This is where I looked: 

<https://github.com/CyrilP/hass-deltadore-tydom-component>

Or am I looking in the wrong place? If that is the case, could you please point me to the correct location?

Thanks 👍 

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]
Oops wrong button, let me try again.

**Comment by CyrilP** [comment]
> Hi there @CyrilP 👋
> 
> I've looked at the upstream repository, but noticed it only contains a blueprint copy, and no actual working integration.
> 
> This is where I looked:
> 
> https://github.com/CyrilP/hass-deltadore-tydom-component
> 
> Or am I looking in the wrong place? If that is the case, could you please point me to the correct location?
> 
> Thanks 👍
> 
> ../Frenck

Hi @frenck 
You are in the right repository, I'm just working in the "new" branch.
I don't want to merge in the main branch yet because it needs some cleanup and there is still a bug when I want to set the boiler on.
It'probably a bit messy, but I'm also learning python

**Comment by frenck** [comment]
No problem! However, there is no use for us to add branding assets to this repository for a non-available integration.

Therefore, I'm going to close this PR for now. Please feel free to re-open this PR when the integration is ready for use.

../Frenck

**Comment by CyrilP** [comment]
Ok, sorry for the inconvenience
I'll reopen as soon as the boiler bug is fixed. I need to do some reverse I think

**Comment by CyrilP** [comment]
Bug is fixed, all is merged on the main branch now.
@frenck, how do I reopen the PR ? I don't see any button for that
